CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # What's New?
 
+## 1.4
+Improvements:
+- Commit text. [PR #pullNumber](https://github.com/microsoft/vscode-cmake-tools/pull/pullNumber) [@gitHubUser](https://github.com/gitHubUser)
+- Documentation updates. [PR #1130](https://github.com/microsoft/vscode-cmake-tools/pull/1130) [@zalava](https://github.com/zalava)
+- Add support for per-folder browse path. [#1073](https://github.com/microsoft/vscode-cmake-tools/issues/1073)
+- Use a shell script to set environment variables for a kit. [#809](https://github.com/microsoft/vscode-cmake-tools/issues/809) [@pisker](https://github.com/pisker)
+- Improvements of the status bar UI. [PR #1200](https://github.com/microsoft/vscode-cmake-tools/pull/1200) [@SchweizS](https://github.com/SchweizS)
+- Add context menu for CMakeLists. [#741](https://github.com/microsoft/vscode-cmake-tools/issues/741) [@SchweizS](https://github.com/SchweizS)
+- Support partial CMake Tools activation for non cmake repos. [PR #1206](https://github.com/microsoft/vscode-cmake-tools/pull/1206)
+- Support ARM IntelliSense modes. [#1155](https://github.com/microsoft/vscode-cmake-tools/issues/1155)
+- Support GNU language standards. [#1208](https://github.com/microsoft/vscode-cmake-tools/issues/1208)
+- Add indication of active workspace to project outline. [#1183](https://github.com/microsoft/vscode-cmake-tools/issues/1183) [@SchweizS](https://github.com/SchweizS)
+
+Bug Fixes:
+- Commit text. [#issueNumber](https://github.com/microsoft/vscode-cmake-tools/issues/issueNumber) [@gitHubUser](https://github.com/gitHubUser)
+- Skip over debugger guessing logic if cmake.debugConfig explicitly sets miDebuggerPath. [#1060](https://github.com/microsoft/vscode-cmake-tools/issues/1060)
+- Normalize all paths sent to CppTools. [#1099](https://github.com/microsoft/vscode-cmake-tools/issues/1099)
+- Add support for Objective-C and Objective-C++. [#1108](https://github.com/microsoft/vscode-cmake-tools/issues/1108) [@marksisson](https://github.com/marksisson)
+- Update extension id. [PR #1116](https://github.com/microsoft/vscode-cmake-tools/pull/1116) [@ChristianS99](https://github.com/ChristianS99)
```

Comment:
This is issue #1045.

Can you update the text to say "Update the configuration provider id" instead?  This is different from the published extension id which we changed in version 1.2.0.